phpBB/phpbb/attachment/upload.php

```diff
@@ -296,8 +296,10 @@ protected function check_attach_quota()
 	 */
 	protected function check_disk_space()
 	{
-		if ($free_space = @disk_free_space($this->phpbb_root_path . $this->config['upload_path']))
+		if (function_exists('disk_free_space'))
 		{
+			$free_space = disk_free_space($this->phpbb_root_path);
```

Comment:
```suggestion
			$free_space = disk_free_space($this->phpbb_root_path . $this->config['upload_path']);
```

Reply:
This is where the function fails there. 
Not sure why we are trying to check such directory?
The space is the space inside the root of the board as `./`

Reply:
@marc1706 this function is `only` being used here https://github.com/phpbb/phpbb/blob/91a5a73dca110aad9f44a90ebfc089517ae351b6/phpBB/phpbb/attachment/upload.php#L195 - I am trying to explain that the check here fails with your proposed change and it is not necessary at all IMHO. The attachment quota is checked by another function as a matter of fact.

Reply:
```suggestion
			$free_space = disk_free_space($this->phpbb_root_path . '/');
```

Reply:
A double slash you mean?

Reply:
Resolved the test failure with changes to the test